src/lib/createElement.js

```diff
@@ -9,6 +9,53 @@
 //    - vNode.children의 각 자식에 대해 createElement를 재귀 호출하여 추가
 
 export function createElement(vNode) {
-  // 여기에 구현하세요
-  return {}
+  console.log(vNode);
+  // 1. vNode가 falsy인 경우
+  if (!vNode) return document.createTextNode("");
```

Comment:
```suggestion
  if (vNode === null || vNode === undefined || typeof vNode === 'boolean') return document.createTextNode("");
```

저도 !vNode로 처리했었는데, 이렇게 하면 숫자 0도 빈 문자열로 렌더링되는 문제가 생긴다고 하더라구요!

저도 솔루션시간에 듣고 바꿨습니당 ㅎㅎ

Reply:
아하 그러네요..! 확인 감사해요:)